gcp_monitoring/gcp_monitoring.yml

```diff
@@ -964,57 +987,73 @@ spec:
-            fieldName: _field
-            visible: true
-          - displayName: instance_name
-            fieldName: instance_name
-            visible: true
-          - displayName: _value
-            fieldName: _value
-            visible: true
-        height: 3
-        kind: Table
-        name: Instance disk r/w ops
-        queries:
-          - query: |-
-                from(bucket: v.bucket)
-                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
-                  |> filter(fn: (r) => r._measurement == "compute.googleapis.com/instance/disk")
-                  |> filter(fn: (r) => r._field == "read_ops_count" or r._field == "write_ops_count")
-                  |> keep(columns: ["_value", "instance_name", "_time", "_field"])
-                //  |> aggregateWindow(every: 1m, fn: sum)
-                //  |> yield(name: "sum")
-                |> sum()
-                |> group(columns: ["table"])
-        tableOptions:
-            verticalTimeAxis: true
-        timeFormat: YYYY-MM-DD HH:mm:ss
+        width: 5
+        xPos: 4
+        yPos: 6
+      - height: 10
+        kind: Markdown
+        name: Name this Cell
+        note: "# LoadBalancing Metrics\n\n**Latency and count of requests, broken
+            down by response status code and proxy names, is the main subject of measurement
+            at this dashboard.**\n\nBased on the `loadbalancing.googleapis.com/https`
+            measurement, sourced from GCP monitoring service. \n\nMetrics are collected
+            via Stackdriver Telegraf plugin. \n\nConfiguration note: in the charts
+            there is a hardcoded path filter in order to select the the URL pattern
+            you can be interested in - in our charts case we call it /platform/. Feel
+            free to adjust the URL regexp pattern according to your needs."
         width: 3
         xPos: 9
-        yPos: 6
-    description: From Telegraf Stackdriver plugin. The dashboard provides deeper insight
-        into the GCP compute services provided by GCP Stackdriver.  Query data from
-        Google Cloud Monitoring (formerly Stackdriver) using the Cloud Monitoring
-        API v3.
-    name: GCP Monitoring - Compute
+    description: From Telegraf Stackdriver plugin. This dashboard monitors network
+        load balancing in the cluster. Query data from Google Cloud Monitoring (formerly
+        Stackdriver) using the Cloud Monitoring API v3.
+    name: GCP Monitoring - HTTPS LoadBalancing
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Telegraf
+metadata:
+    name: distracted-jang-ca9005
+spec:
+    associations:
+      - kind: Label
+        name: sweet-nobel-8a9003
+    config: |
+        [agent]
+          interval = "1m"
+          flush_interval = "10s"
+          metric_buffer_limit = 50000
+          debug = true
+          omit_hostname = false
+        [[outputs.influxdb_v2]]
+          urls = ["https://us-central1-1.gcp.cloud2.influxdata.com"]
```

Comment:
Since the other settings here are environment variables, you might want to make this $INFLUX_HOST